frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -723,7 +725,7 @@ private void loadAndStoreStickyProjectGroup(PortletRequest request, User user, M
             if (!isNullOrEmpty(parameter)) {
                 Set<String> values = CommonUtils.splitToSet(parameter);
                 if (filteredField.equals(Project._Fields.NAME)) {
-                    values = values.stream().map(v -> v + "*").collect(Collectors.toSet());
+                    values = values.stream().map(v -> sanitizeQueryInput(v) + "*").collect(Collectors.toSet());
```

Comment:
I've counted 9 occurences of `sanitizeQueryInput(something) + "*")`.  I think this warrants extracting a method with a descriptive name.

Reply:
created methods `prepareWilcardQuery` and `prepareFuzzyQuery`